packages/react-components/src/components/Survey/TagsSurvey/TagsSurvey.tsx

```diff
@@ -118,10 +125,10 @@ export const _TagsSurvey = (props: _TagsSurveyProps): JSX.Element => {
   }, [callIssuesToTag]);
 
   const onChange = React.useCallback(
-    (issueCategory: string, checked: boolean, issue?: string): void => {
+    (issueCategory: keyof SurveyIssues, checked: boolean, issue?: string): void => {
       if (checked) {
         if (issue) {
-          setSelectedTags((prevState) => {
+          setSelectedTags((prevState: { [key: string]: { score: number; issues: string[] } }) => {
```

Comment:
The original definition of setSelectedTags should be typed instead:

```
- const [selectedTags, setSelectedTags] = useState({});
+ const [selectedTags, setSelectedTags] = useState<STRONG_TYPE_HERE>({});
```

Reply:
Also where does `{ [key: string]: { score: number; issues: string[] } }` come from? Is this an existing type somewhere? Maybe `_CallRating`?

Reply:
With help from Caroline we resolved this particular concern! Great teamwork James/Caroline/Myself